homeassistant/components/tuya/models.py

```diff
@@ -173,6 +172,12 @@ def from_json(cls, dpcode: DPCode, data: str) -> EnumTypeData | None:
         return cls(dpcode, **parsed)
 
 
+_TYPE_DATA_MAPPINGS: dict[DPType, type[TypeData]] = {
+    DPType.ENUM: EnumTypeData,
+    DPType.INTEGER: IntegerTypeData,
+}
```

Comment:
This will make it easier to add proper support for `DPType.BITMAP` in a follow-up PR